.sphinx/lxddoc/lxddoc.txt

```diff
@@ -0,0 +1,43 @@
+---
+orphan: true
+---
+
+```{config:option} scheduler.instance cluster
+:liveupdate: `yes`
+:shortdesc: Possible values are all, manual and group. See Automatic placement of instances for more information.
+:condition: container
+:default: `all`
+:type: integer
+
+ The cluster config keys
+```
+
+```{config:option} user.* cluster
+:shortdesc: Free form user key/value storage (can be used in search).
+:condition: container
+:default: -
+:type: string
+:liveupdate: `yes`
+
+This is the real long desc.
+
+With two paragraphs.
+
+And a list:
+
+- Item
+- Item
+- Item
+
+And a table:
+
+Key                                 | Type      | Scope     | Default                                          | Description
+:--                                 | :---      | :----     | :------                                          | :----------
+`acme.agree_tos`                    | bool      | global    | `false`                                          | Agree to ACME terms of service
+`acme.ca_url`                       | string    | global    | `https://acme-v02.api.letsencrypt.org/directory` | URL to the directory resource of the ACME service
+`acme.domain`                       | string    | global    | -                                                | Domain for which the certificate is issued
+`acme.email`                        | string    | global    | -                                                | Email address used for the account registration
+
+
```

Comment:
There shouldn't be empty lines in here (unless they come explicitly from the comment).